package api.usecases.task.finish

class TaskAlreadyFinishedException: TaskFinishException, Exception()
